addons/account/models/account_move.py

```diff
@@ -3148,6 +3152,22 @@ def field_will_change(line, field_name):
                 return not line[field.currency_field].is_zero(line[field_name] - vals[field_name])
             return line[field_name] != vals[field_name]
 
+        def get_tracking_field_values(field, value):
+            if field.type == 'date':
+                return value and format_date(self.env, fields.Date.from_string(value)) or ''
+            elif field.type in ('one2many', 'many2many', 'many2one'):
+                return value and value.name_get()[0][1] or ''
+            else:
+                return value or ''
+
+        def get_tracking_field_string(move, fields):
+            ARROW_RIGHT = '<span aria-label="Changed" class="fa fa-long-arrow-right" role="img" title="Changed"></span>'
```

Comment:
translatability + message should be improved to include the Id of the line that changed